### PR TITLE
Update documents

### PR DESCRIPTION
**Pull request by RoboticsYY**
This PR intends to update the tutorials about hand-eye calibration, draw_x and fixed_position_pick apps.

### CODE REVIEW

File: grasp_tutorials/doc/draw_x.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+.. _UR5: https://www.universal-robots.com/products/ur5-robot
+
+.. _ROS2 Dashing: https://index.ros.org/doc/ros2/Installation/Dashing/Linux-Install-Debians/
+
+.. _robot_interface: https://github.com/intel/ros2_grasp_library/tree/master/grasp_utils/robot_interface
+
+Download and Build the Example Code
+------------------------------------
+
+Within your catkin workspace, download and compile the example code:
```
Comment (sharronliu) [inline grasp_tutorials/doc/draw_x.rst]: catkin >> ros2?

File: grasp_tutorials/doc/fixed_position_pick.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+.. _ROS2 Dashing: https://index.ros.org/doc/ros2/Installation/Dashing/Linux-Install-Debians/
+
+.. _robot_interface: https://github.com/intel/ros2_grasp_library/tree/master/grasp_utils/robot_interface
+
+.. _Robot Gripper: https://www.universal-robots.com/plus/end-effectors/hitbot-electric-gripper
+
+Download and Build the Example Code
+------------------------------------
+
+Within your catkin workspace, download and compile the example code:
```
Comment (sharronliu) [inline grasp_tutorials/doc/fixed_position_pick.rst]: catkin >> ros2?

File: grasp_tutorials/doc/draw_x.rst (hunk 1 above)
Comment (RoboticsYY) [reply grasp_tutorials/doc/draw_x.rst]: Done. Name corrected.

File: grasp_tutorials/doc/fixed_position_pick.rst (hunk 2 above)
Comment (RoboticsYY) [reply grasp_tutorials/doc/fixed_position_pick.rst]: Done. Name corrected.

**Pull request merged**